flatcar-postinst

```diff
@@ -446,3 +446,6 @@ call_cgpt show "${INSTALL_DEV}"
 
 cat "${INSTALL_MNT}/share/flatcar/release"
 echo "Setup ${INSTALL_LABEL} (${INSTALL_DEV}) for next boot."
+
+# Create the Ubuntu-compatible /run/reboot-required flag file for kured
+touch /run/reboot-required
```

Comment:
Can/Should this be tested somehow?

Reply:
At least I want to run a test with using the OEM hook that gets called earlier. I'll also try to run kured manually on a built image and will merge when done.

Reply:
Running the kured container without K8s doesn't really work easily but I checked that the file gets created.